### ISSUE TITLE
Use Bytes crate to handle underlying storage.

### ISSUE THREAD

**Issue by djg** [djg/peek-poke#3]
Should this crate use `Bytes` (https://github.com/carllerche/bytes) to manage the underlying storage?

**Comment by djg** [comment]
Any comments @Gankro, @jrmuizel?

**Comment by jrmuizel** [comment]
Maybe, though I don't see any immediate benefit.

**Comment by Gankra** [comment]
I agree that I don't see what this does for our usecase (notably we don't care about endianess).

**Comment by djg** [comment]
That it provides a tested, safe interface to accessing bytes from a Vec<u8> instead of using raw pointers?
